src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -39,6 +39,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.3.1" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />

```

Comment:
Is this still needed? Can it be removed?

Reply:
removed.